pkg/tracer/offsetguess.go

```diff
@@ -302,18 +316,18 @@ func guess(b *elf.Module) error {
 	}
 
 	listenCompleted := make(chan freePort)
-	closeListen := make(chan struct{})
+	acceptChan := make(chan struct{})
 
 	var listenPort uint16
-	go listenV4(listenCompleted, closeListen)
+	go listenV4(listenCompleted, acceptChan)
```

Comment:
I should have noticed this earlier but, how about splitting the listening and accepting parts? (Making the listening part blocking) Using a channel just to immediately block on it just complicates things